releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -59,6 +59,10 @@ As an example, `IContextualFeatureFilter<IAccountContext>` requires a context th
 
 With the introduction of `IContextualFeatureFilter` there are now two types of feature filters including `IFeatureFilter`. The two types of feature filters both inherit `IFeatureFilterMetadata`. `IFeatureFilterMetadata` is a marker interface and does not actually provide any feature filtering capabilities. It is used as the new parameter type for `IFeatureManagementBuilder.AddFeatureFilter`.
 
+**Feature Settings Cache**
```

Comment:
Is this for "Performance Improvement"? If so, a customer may make more sense from that title.

Reply:
Yes, good point. Updated.